src/themes/teams/utils/index.ts

```diff
@@ -0,0 +1,3 @@
+import { pxToRem as basePxToRem } from '../../../lib'
+
+export const pxToRem = (sizeInPx: number) => basePxToRem(sizeInPx, 14)
```

Comment:
Can we avoid there a magic number and define `14` as a separate `const`? 😺 

```suggestion
/* Teams use 14px as a base font size. */
const baseFontSize = 14

export const pxToRem = (sizeInPx: number) => basePxToRem(sizeInPx, baseFontSize)
```